### DISCUSSION TITLE
Thanks for giving me Pull Shark tag

### DISCUSSION THREAD

**Discussion by htouqeer938** [community/community#46952] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

**Thanks for giving me Pull Shark tag**

**Comment by jge162** [comment]
Nice.

**Comment by Ntshangase** [comment]
If you want more Tags/Badges you should check out [https://github.com/Schweinepriester/github-profile-achievements](url) the link would show you how to earn more.

**Reply by diannaowa** [reply]
Sorry, the link is 404.

**Reply by Ntshangase** [reply]
I have just checked it sorry, A working link= https://github.com/Schweinepriester/github-profile-achievements

I also have an Open Source project you can use it if you want to style your GitHub profile, 
link= https://github.com/Ntshangase/GitHub-Profile-Builder

**Comment by Faiqlatheef** [comment]
After answering two discussions, You will get Galaxy Brain badge
And create 2 or more branches and make pull request and merge it You will get one
More details here:
https://github.com/Schweinepriester/github-profile-achievements
